packages/tocco-ui/src/Panel/PanelHeaderFooter.spec.js

```diff
@@ -0,0 +1,84 @@
+
+import React from 'react'
+import {shallow} from 'enzyme'
+
+import PanelHeaderFooter from './PanelHeaderFooter'
+import Button from '../Button'
+import StyledPanelHeaderFooter from './StyledPanelHeaderFooter'
+
+describe('tocco-ui', function() {
```

Comment:
should there be an additional namespace `Panel` here so that it reflects the actual component structure?
--> some intermediate namespaces missing in some more tests too

Reply:
Adding namespaces is not mandatory, because the component is imported directly and not resolved from index.js.

I admit it would enhance clarity if any example and unit test in tocco-ui would use dot notation (e.g. <Panel.Header> instead of <PanelHeaderFooter>). Such correction is a good task for the next quality day.

Reply:
hm rzueger is right. 

`describe('Panel', ....`

is missing. We we have the whole folder structure in our spec files. It doenst matter how a component is imported.

Reply:
I missed the point. Thanks for being persistent.